ios/Classes/FlutterArkitView+Handlers.swift

```diff
@@ -225,4 +225,11 @@ extension FlutterArkitView {
         }
    }
 
+   func onGetSnapshot(_ result:FlutterResult) {
```

Comment:
I don't think this TODO comment is actual as you already converting the result. The next line is describing what is going on here instead of why this is needed. Usually such comments might safely be removed. The only part that is not in this code is that `FlutterStandardTypedData` is `UInt8List` in Dart, so you may put the comment near line 232 and shorten it just to
```swift
// UInt8List in Dart
```
or (which I prefer) - remove this comment at all.

You're forcing pngData to be not null which might not be true. I propose to extend your implementation like that:
```swift
   func onGetSnapshot(_ result:FlutterResult) {
        let snapshotImage = sceneView.snapshot()
        if let bytes = snapshotImage.pngData() {
            let data = FlutterStandardTypedData(bytes:bytes)
            result(data)
        } else {
            result(nil)
        }
    }
```

Reply:
Yes. Was thinking this last night in bed (like you do). I was in a rush to get the app working but definitely some null safety required. I'll make some amends. Would also like to look at other features that may need addressing. 👍